gk/main.c

```diff
@@ -918,20 +941,19 @@ process_pkts_front(uint8_t port_front, uint8_t port_back,
 				 * Notice that one needs to update
 				 * the Ethernet header.
 				 *
-				 * TODO Add sequential lock to deal with the
+				 * Use sequential lock to deal with the
 				 * concurrency between GK and LLS on the cached
 				 * Ethernet header.
 				 */
```

Comment:
Given that the sequential lock is handled inside `pkt_copy_cached_eth_header()`, we can drop this last paragraph of the comment.